tests/test_likelihoods.py

```diff
@@ -97,3 +98,21 @@ def test_no_inf_values_v(data_ddm, shared_params):
         assert np.all(
             np.isfinite(logp.eval())
         ), f"log_pdf_sv() returned non-finite values for v = {v}."
+
+
+def test_bbox(data_ddm):
```

Comment:
for these tests, it might make sense to take a sample of a bunch of parameter vectors instead of one fixed such vector.

Reply:
Let's fix these tests in a different PR. I feel like there is a lot of improvements that we can do to these tests to make them much faster